docs/brokerpak-intro.md

```diff
@@ -1,117 +1,65 @@
-# Creating Custom Services (UNRELEASED)
+# Brokerpak introduction
 
-This documentation is for an **UNRELEASED** upcoming feature for the GCP Service Broker and should not be considered complete.
+This document will explain brokerpaks, what they are, the problems they solve, how they them, and best practices around developing, maintaining, and using paks.
 
-## Variable resolution
+## What is a brokerpak?
 
-The variables fed into your Terraform services file are resolved in the following order:
+To understand the how and why of brokerpaks, it's first important to understand the kind of problems that service brokers solve.
 
-* Variables defined in your `computed_variables` JSON list.
-* (Only for Provision) Variables defined by the selected service plan in its `service_properties` map.
-* User defined variables (in `provision_input_variables` or `bind_input_variables`)
-* Operator default variables loaded from the environment.
-* Default variables (in `provision_input_variables` or `bind_input_variables`).
+### A quick aside about service brokers
 
-Note that the order the variables are combined in code is slightly different.
+A _service broker_ provides an interface between an _application provider_ (e.g. GCP, Azure or AWS), and an _application platform_ (e.g. Kubernetes or Cloud Foundry).
```

Comment:
Is "application provider" the right description here? What about "service" or "resource" provider?